fairlearn/datasets/_fetch_acs_income.py

```diff
@@ -86,12 +86,17 @@ def fetch_acs_income(
             Array of ordered feature names used in the dataset.
         DESCR : string
             Description of the ACSIncome dataset.
-
-    (data, target) : tuple of (numpy.ndarray, numpy.ndarray)
-        if ``return_X_y`` is True and ``as_frame`` is False
-
-    (data, target) : tuple of (pandas.DataFrame, pandas.Series)
-        if ``return_X_y`` is True and ``as_frame`` is True
+        categories : dict or None
+            Maps each categorical feature name to a list of values, such that the
+            value encoded as i is ith in the list. If ``as_frame`` is True, this is None.
+        frame : pandas DataFrame
+            Only present when ``as_frame`` is True. DataFrame with ``data`` and ``target``.
+        
+    (data, target) : tuple if ``return_X_y`` is True
+
+    Notes
+    ----------
+    We follow the convention of `sklearn.datasets.fetch_openml` to document this function.
```

Comment:
```suggestion
    We follow the convention of :func:`sklearn.datasets.fetch_openml` to document this function.
```
I believe this little change should generate link to the `fetch_openml` API docs on the scikit-learn page. If not, we can try adding `~` before `sklearn` just like with `Bunch` earlier in this file.